### PR TITLE
use icons for status

### LINKED ISSUE #1807: Fix Table examples where status "icons" are used in cells. Use the shaped StatusIcon instead of rounded Box for accessibility.
Currently, our implementation is:

Screen Shot 2021-08-26 at 11 27 18 AM

The corresponding label to the colored shape does satisfy accessibility because it doesn't solely rely on color. However, the direction we have been going is to use differentiated shapes + colors + labels as represented in the Notifications/Status work.

- [x] Use status icons in this [Table example](https://deploy-preview-1817--keen-mayer-a86c8b.netlify.app/components/table#sorting)
- [x] Check if anywhere else in the site needs similar update
- [x] Include entry for What's New

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR adds the icons for the status 
#### Where should the reviewer start?
table/TableSortable
#### What testing has been done on this PR?
locally 
#### How should this be manually tested?
look at example for the sortable should be the same
#### Any background context you want to provide?
We want to use  differentiated shapes + colors + labels as represented in the Notifications/Status
#### What are the relevant issues?
closes #1807 
#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: aries-site/src/examples/components/table/TableSortable.js [hunk 1]
```diff
@@ -103,12 +108,7 @@ const columns = [
     header: 'Status',
     render: datum => (
       <Box direction="row" align="center" gap="xsmall">
-        <Box
-          height="12px"
-          width="12px"
-          background={datum.status.color}
-          round="100%"
-        />
+        <datum.status.icon color={datum.status.color} />
```
Comment (halocline) [inline aries-site/src/examples/components/table/TableSortable.js:111]:
```suggestion
        <datum.status.icon color={datum.status.color} size="small" />
```

**Comment by halocline** [comment]
I also added a [checklist of items to the issue](https://github.com/grommet/hpe-design-system/issues/1807) before we close it out.

**Review by taysea — APPROVED** [review]
LGTM. And looks like you took care of all the checklist items mentioned above ^

**Comment by britt6612** [comment]
There was 
https://design-system.hpe.design/templates/filtering#more-examples-of-filtering
https://design-system.hpe.design/templates/filtering#filtering-with-select

Theres 2 examples but their not really a status so I didn't change them to use the Status icons

any thoughts?

**Comment by halocline** [comment]
> There was
> https://design-system.hpe.design/templates/filtering#more-examples-of-filtering
> https://design-system.hpe.design/templates/filtering#filtering-with-select
> 
> Theres 2 examples but their not really a status so I didn't change them to use the Status icons
> 
> any thoughts?

Good finds. Though not "status," I'd say they are "status indicators" and should be swapped out for consistency.

**Comment by britt6612** [comment]
> > There was
> > https://design-system.hpe.design/templates/filtering#more-examples-of-filtering
> > https://design-system.hpe.design/templates/filtering#filtering-with-select
> > Theres 2 examples but their not really a status so I didn't change them to use the Status icons
> > any thoughts?
> 
> Good finds. Though not "status," I'd say they are "status indicators" and should be swapped out for consistency.

They are swapped! This was everything that contained any kind of status  that I saw

File: aries-site/src/pages/whats-new.mdx [hunk 2]
```diff
@@ -51,6 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## August 30 - September 3, 2021
+
+- Published [Notifications page](https://design-system.hpe.design/components/notification) 
+show the first variant of using [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:57]: Either link to the "first variant" using "/components/notification#toast-notification" or separate.

```suggestion
- Published [Notifications page](/components/notification).
- Published [Toast Notifications](/templates/toast-notifications).
```

```suggestion
- Published [Notifications page](/components/notification) 
showing the [first variant](/components/notification#toast-notification) using [Toast Notifications](/templates/toast-notifications).
```

File: aries-site/src/pages/whats-new.mdx [hunk 3]
```diff
@@ -51,6 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+along with an example.
+- Publshed [Status Indicators](https://design-system.hpe.design/templates/status-indicator) template page.
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:59]:
```suggestion
- Published [Status Indicators](https://design-system.hpe.design/templates/status-indicator) template page.
```

File: aries-site/src/pages/whats-new.mdx [hunk 4]
```diff
@@ -51,6 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- Updated table example to use the new Status Indicators guidance.
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:60]:
```suggestion
- Updated all examples referencing "status" to apply the new [Status Indicators guidance](/templates/status-indicator#what-makes-up-a-status-indicator).
```

**Review by halocline — COMMENTED** [review]
Minor "What's New" cleanups. Everything else looks 🎉

File: aries-site/src/pages/whats-new.mdx [hunk 5]
```diff
@@ -51,6 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## August 30 - September 3, 2021
+
+- Published [Notifications page](/components/notification).
+- Published [Toast Notifications](/templates/toast-notifications).
+along with an example.
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:58]:
```suggestion
- Published [Toast Notifications](/templates/toast-notifications).
```

Comment (halocline) [reply aries-site/src/pages/whats-new.mdx:58]: Let me know when either the "extra period" or dangling sentence is removed. I will approve and merge after that.

**Review by halocline — APPROVED** [review]

**Pull request merged**